macaw-x86-cli/test-data/byte-id.cbl

```diff
@@ -0,0 +1,10 @@
+(defun @byte_id ((x (Bitvector 8))) (Bitvector 8)
+  (start start:
+    (return x)))
+
+(defun @main () Unit
+  (start start:
+    (let input (fresh (Bitvector 8)))
+    (let input_back (funcall @byte_id input))
+    (assert! (equal? input input_back) "byte_id test")
+    (return ())))
```

Comment:
Having a basic test case like this is fine, but I do think it would be worth also adding a test case that demonstrates a capability that is unique to `macaw-x86-cli`.

Reply:
Agreed, it's just a bit tough at the moment as the only x86_64-specific syntax we support is `X86Regs`, `{g,s}et-reg`, and the individual register names. To use any of that, we'd have to construct an `X86Regs`. This would presumably involve initializing all of its fields, which are many. `fresh` also doesn't seem to cut it:
```
At test-data/get-rax.cbl:3:17, expected Crucible atom literal but got fresh
```
Mind if I leave this to a follow-up issue?

Reply:
> Mind if I leave this to a follow-up issue?

Sure.

Reply:
https://github.com/GaloisInc/macaw/issues/428